readthedocs/projects/forms.py

```diff
@@ -272,6 +273,69 @@ def __init__(self, *args, **kwargs):
         else:
             self.fields['default_version'].widget.attrs['readonly'] = True
 
+        self.setup_external_builds_option()
+
+    def setup_external_builds_option(self):
+        """Disable the external builds option if the project doesn't meet the requirements."""
+        msg = None
+        url = reverse("projects_integrations", args=[self.instance.slug])
+        integrations = list(self.instance.integrations.all())
+        if not self.has_supported_integration(integrations):
+            msg = _(
+                "To build from pull requests you need a "
+                'GitHub or GitLab <a href="{url}">integration</a>.'
```

Comment:
Where are we populating this `{url}` in the string? I didn't find it.

Maybe it's missing an `f` at the beginning?

```suggestion
                f'GitHub or GitLab <a href="{url}">integration</a>.'
```